test_conformance/subgroups/test_workgroup.cpp

```diff
@@ -825,12 +821,18 @@ cl_uint4 generate_bit_mask(cl_uint subgroup_local_id, std::string mask_type) {
     if (mask_type == "eq") {
         if (subgroup_local_id < 32) {
             first_el = set_bit(1, 0, pos);
+            second_el = 0xFFFFFFFF;
```

Comment:
this looks strange. We set 1 in the position where subgroup_local_id is detected, why 1 in other vector components ?

Reply:
all other changes are ok

Reply:
Fixed